src/data/python-quiz.ts

```diff
@@ -1004,6 +1004,24 @@ const pythonQuiz = [
     Explanation: "If you don't specify the start index, the default start index of 0 is used.",
     Link: "https://www.freecodecamp.org/news/python-range-function-explained-with-code-examples/",
   },
+  {
+    Question: "Which data structure is used for implementation of recursive algorithm?",
+    Answer: "Stack",
+    Distractor1: "Queue",
+    Distractor2: "Stack",
```

Comment:
Distractors can't be the same as the correct answer
Please choose another option.